ultralight-java-databind/src/main/java/net/labymedia/ultralight/utils/JavascriptUtils.java

```diff
@@ -0,0 +1,106 @@
+        if (javaObject == JavascriptType.UNDEFINED) return context.makeUndefined();
+        if (javaObject == null) return context.makeNull();
+
+        if (javaClass == Short.class
+                || javaClass == Integer.class
+                || javaClass == Long.class
+                || javaClass == Float.class
+                || javaClass == Double.class
+                || javaClass == Byte.class
+                || javaClass == short.class
+                || javaClass == int.class
+                || javaClass == long.class
+                || javaClass == float.class
+                || javaClass == double.class
+                || javaClass == byte.class) return context.makeNumber((double) javaObject);
+        if (javaClass == Character.class) return context.makeString(String.valueOf((char) javaObject));
+        if (javaClass == Boolean.class) return context.makeBoolean((boolean) javaObject);
+        if (javaClass == String.class) return context.makeString((String) javaObject);
+        if (javaClass == Date.class) {
+            Date date = (Date) javaObject;
+
+            return context.makeDate(
+                context.makeNumber(date.getYear() + 1900),
+                context.makeNumber(date.getMonth()),
+                context.makeNumber(date.getDate()),
+                context.makeNumber(date.getHours()),
+                context.makeNumber(date.getMinutes()),
+                context.makeNumber(date.getSeconds())
+            );
+        }
+        if (javaClass == char.class) return context.makeString(String.valueOf((char) javaObject));
+        if (javaClass == boolean.class) return context.makeBoolean((boolean) javaObject);
+        if (javaClass == JavascriptClass.class) return context.makeObject((JavascriptClass) javaObject);
+
+        try {
+            if (javaClass == Class.class) {
+                return context.makeObject(new UltralightDatabind(UltralightDatabindConfiguration.builder().build()).toJavascript(javaClass));
+            } else {
+                return context.makeObject(new UltralightDatabind(UltralightDatabindConfiguration.builder().build()).toJavascript(javaClass), javaObject);
+            }
+        } catch (IllegalAccessException exception) {
+            exception.printStackTrace();
+            throw new IllegalArgumentException("Java object could not be contained in a javascript value", exception);
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    public static <T> GenericValueContainer<T> fromJavascriptValue(final JavascriptValue value) {
+        if (value.isObject()) {
+            if (value.isDate()) {
+                try {
+                    return (GenericValueContainer<T>) new GenericValueContainer<>(JAVASCRIPT_DATE_FORMAT.parse(value.toString()));
+                } catch (ParseException exception) {
+                    exception.printStackTrace();
```

Comment:
Don't randomly print exceptions, rather re-throw the exception wrapped with an unchecked exception. (Such as ``IllegalArgumentException``)